lab-phelan/lib/error-middleware.js

```diff
@@ -0,0 +1,46 @@
+'use strict';
+
+const logger = require('./logger');
+
+module.exports = (error,request,response/*,next*/) => {
+
+
```

Comment:
at maximum you should only have one line of whitespace between code, for best practice. also please remove the commented out next from line 5 - you aren't using it so it should be deleted.